src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1396 @@
+    int32_t select = 0;
+    bool allmsupport = false;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+
+    UT_LOG_INFO(" \n================Please Select Inputs==================\n");
+    UT_LOG_INFO("\n*******Enter the port number to select*******\n");
+    
+    readInput(&select);
+    if(select <= 0 || select > (numInputPorts-1))
+    {
+       UT_LOG_ERROR("\nInvalid port selected\n");
+       return;
+    }
+
+    port = (select-1);
+
+    UT_LOG_INFO("\n*******Enter the allmsupport to select*******\n\t"
+                     "\tAcceptable inputs are:\n\t"
+                     "\ttrue (i.e 1)\n\t"
+                     "\tfalse(i.e 0)\n\t"
+                "******************************************\n");
+
+    readInput(&select);
+    if(select < 0 || select > 1)
+    {
+       UT_LOG_ERROR("\nInvalid allmsupport value selected\n");
+       return;
+    }
+  
+    allmsupport = (bool)select;
+
+    UT_LOG_INFO("Calling dsSetEdid2AllmSupport IN:port:[%s]:[%d] ,IN:allmsupport:[%s]",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), (port+1),
+                 UT_Control_GetMapString(bool_mapTable, allmsupport));
+
+    ret = dsSetEdid2AllmSupport(port, allmsupport);
+
+    UT_LOG_INFO("Calling dsSetEdid2AllmSupport IN:port:[%s]:[%d] ,IN:allmsupport:[%s]",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), (port+1),
+                 UT_Control_GetMapString(bool_mapTable, allmsupport));
+    DS_ASSERT(ret == dsERR_NONE);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get allmsupport.
+*
+* This test function gets allmsupport of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 017@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_edid2allmsupport(void)
+{
+    gTestID = 17;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_MAX;
+    int32_t select = 0;
+    bool allmsupport = false;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+
+    UT_LOG_INFO(" \n================Please Select Inputs==================\n");
+    UT_LOG_INFO(" \n*******Enter the port number to select*******\n");
+    
+    readInput(&select);
+    if(select <= 0 || select > (numInputPorts -1))
+    {
+        UT_LOG_ERROR("\nInvalid port selected\n");
+        return;
+    }
+
+    port = (select - 1);
+
+    UT_LOG_INFO("Calling dsSetEdid2AllmSupport IN:port:[%s]:[%d], OUT:allmsupport:[ ]",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), (port+1)),
+
+    ret = dsGetEdid2AllmSupport(port, &allmsupport);
+
+    UT_LOG_INFO("Result dsSetEdid2AllmSupport IN:port:[%s]:[%d], OUT:allmsupport:[%s]",
```

Comment:
Typo mismatch **"dsSetEdid2AllmSupport"** in API name in log print for both Calling and Result prints. Change to **dsGetEdid2AllmSupport**

Reply:
done